src/content/wiki/workers/miner.mdoc

```diff
@@ -30,6 +34,7 @@ The level of the Mine will dictate how deep the Miner will mine:
 | 4              | Bedrock       |
 | 5              | "             |
 {% /version %}
+{% /version %}
```

Comment:
Try to avoid nested version blocks if possible, this can very easily mess something up if the ranges are different

Reply:
Is it possible to specify a version range? One of those tables should be 1.18+ and 1.20-, but I did not know another way to specify that

Reply:
Version ranges are not supported, no, could look into that for the future, so I guess it's fine for now then